packages/addons/_tests/storybook/test.ts

```diff
@@ -9,6 +11,11 @@ const { test, variants, prepareServer } = setupTest({ storybook, eslint });
 
 let port = 6006;
 
+beforeAll(() => {
+	// prefetch the storybook cli to reduce fetching errors in tests
+	execSync('pnpx create-storybook@latest --version');
+});
```

Comment:
adding the prefetch to the workflow file instead of directly in the test code was intentional in #622 so that we're not prefetching storybook _every time we run `pnpm test` locally_. if we're going to continue with this setup, we should at least wrap it in a `process.env.CI` check so that it only prefetches in CI

Reply:
I did understand that. Background here is https://github.com/sveltejs/svelte-ecosystem-ci/pull/32, which requires us to also add that prefetching in ecosystem-ci. And remembering that / keeping that in sync is probably going to be hard.

I'm fine by adding a CI check. 

> every time we run pnpm test locally
Only true if you always run the addons project. I rarely run them locally, as they are super computationally intensive and my laptop just doesn't like them. But that might be a me problem.

Reply:
> I'm fine by adding a CI check.

should be good to merge then once that's added!